README.md

```diff
@@ -1,2 +1,68 @@
-# pcb
-Monolithic repository containing QFSAE PCB designs.
+# QFSAE PCB Projects
+
+This repository stores the various PCB designs made by the QFSAE team as well as useful reference designs.
+
+# Altium
+
+The QFSAE team designs its boards using Altium. If you do not have Altium installed, Please contact Logan or Brent to obtain the appropriate licensing information to get Altium up and running on your computer. In regards to manufacturing, the team most often makes use of [JLCPCB](https://jlcpcb.com/). Traditionally, PCBs made by QFSAE members have been populated by hand in house. However, due to the restrictions imposed on team operation by the ongoing COVID-19 pandemic, it has become of utmost importance to create an Altium workflow which allows us to outsource our board assembly in an efficient manner. JLCPCB offers an economical assembly service for surface mount components with the following caveats.
+
+- All surface mount components must be on one side of the board
+- Through hole parts will not be soldered
+- The components must be from the JLCPCB parts that they keep in stock at their manufacturing facility.
+- The part being populated on the board must be within the size and clearance constraints given by JLC PCB.
+
+In order to meet these manufacturing conditions, we must restrain the SMD parts we use to those offered by JLCPCB and use Altium design rules that reflect the manufacturing capabilities of JLCPCB when checking our PCB layouts. The parts library is quite vast with over 30,000 components. With that being said, there will still be cases where a custom footprint or one from another library must be used. In that case, check with another team member to verify that it can be hand soldered properly. The sections below provide instructions on importing the JLC parts libraries, which are included in this repository as well as design rules.
+
+## Library Setup
+The JLC PCB Altium component libraries are located in the libraries folder of this repository. Import the libraries using the instructions [here](https://www.altium.com/documentation/altium-designer/integratedlibrary-dlg-addremovelibrariesformavailable-file-based-libraries-ad). Your final available libraries panel should like the photo below. Be sure to also keep the Altium default libraries and search for parts when making your schematic through the integrated library and not the schematic libraries. 
+
+![Altium Panel](https://i.imgur.com/d9237hu.png)
+
+## Design Rule Configuration
+
+The design rules you import will depend on the layer count and copper density you want to use (JLCPCB offers 1oz and 2oz). The `.RUL` Altium design rule files can be found in `pcb/libraries/jlcpcb-design-rules-stackups/design-rules/altium`. The instructions for importing the design rules into your project can be found [here](https://www.altium.com/cn/documentation/altium-designer/constraining-the-design-design-rules-ad?version=18.1).
```

Comment:
URL links to chinese region altium docs, use instead 
https://www.altium.com/documentation/altium-designer/constraining-the-design-design-rules-ad

Reply:
Actually this would be even better, cuts straight to the right section 
https://www.altium.com/documentation/altium-designer/constraining-the-design-design-rules-ad#!exporting-and-importing-rules

Reply:
fixed!